agent/03_agent_deployment.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+LOGDIR=${SCRIPTDIR}/logs
+source $SCRIPTDIR/logging.sh
```

Comment:
the scriptdir could have spaces, better double quote it

Reply:
Looking forward to the Ansible version already.